app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/topsites/TopSiteItemViewHolder.kt

```diff
@@ -122,5 +126,15 @@ class TopSiteItemMenu(
                 onItemTapped.invoke(Item.RemoveTopSite)
             }
         )
+
+        if (isPinnedSite) {
```

Comment:
I put rename before remove, I think it makes more sense. Also, given that there are now more menu items, I highlighted the remove item in red (I don't remember where, but I copied it from somewhere already implemented in the UI, maybe collection removing?):

```SimpleBrowserMenuItem(..., textColorResource = ThemeManager.resolveAttribute(R.attr.destructive, context)```